pkg/asset/agent/manifests/staticnetworkconfig/generator.go

```diff
@@ -219,20 +220,26 @@ func (s *staticNetworkConfigGenerator) formatNMConnection(nmConnection string) (
 func (s *staticNetworkConfigGenerator) ValidateStaticConfigParams(ctx context.Context, staticNetworkConfig []*models.HostStaticNetworkConfig) error {
 	var err *multierror.Error
 	for i, hostConfig := range staticNetworkConfig {
-		err = multierror.Append(err, s.validateMacInterfaceName(i, hostConfig.MacInterfaceMap))
+		err = multierror.Append(err, s.validateMacInterfaceTable(i, hostConfig.MacInterfaceMap))
 		if validateErr := s.ValidateNMStateYaml(ctx, hostConfig.NetworkYaml); validateErr != nil {
 			err = multierror.Append(err, fmt.Errorf("failed to validate network yaml for host %d, %w", i, validateErr))
 		}
 	}
 	return err.ErrorOrNil()
 }
 
-func (s *staticNetworkConfigGenerator) validateMacInterfaceName(hostIdx int, macInterfaceMap models.MacInterfaceMap) error {
+func (s *staticNetworkConfigGenerator) validateMacInterfaceTable(hostIdx int, macInterfaceMap models.MacInterfaceMap) error {
+	if len(macInterfaceMap) == 0 {
+		return fmt.Errorf("at least one interface for host %d must be provided", hostIdx)
```

Comment:
Just wanted to check my understanding. We don't validate the interfaces also exist in the raw NMStateConfig, correct?

Reply:
That's correct, this is just for checking the interfaces in the MAC Interface table.